modules/tinymce/src/plugins/link/main/ts/core/Actions.ts

```diff
@@ -76,18 +79,27 @@ const setupGotoLinks = (editor: Editor) => {
   });
 };
 
-const toggleActiveState = (editor: Editor) => function (api) {
-  const nodeChangeHandler = (e) => api.setActive(!editor.mode.isReadOnly() && !!Utils.getAnchorElement(editor, e.element));
-  editor.on('NodeChange', nodeChangeHandler);
-  return () => editor.off('NodeChange', nodeChangeHandler);
+const toggleState = (editor: Editor, toggler: (e: NodeChangeEvent) => void) => {
+  editor.on('NodeChange', toggler);
+  return () => editor.off('NodeChange', toggler);
 };
 
-const toggleEnabledState = (editor: Editor) => function (api) {
+const toggleActiveState = (editor: Editor) => (api: Toolbar.ToolbarToggleButtonInstanceApi | Menu.ToggleMenuItemInstanceApi) =>
+  toggleState(editor, () => {
```

Comment:
I'm surprised ESLint isn't complaining about the `{}` wrapping one line of code?

Reply:
Why should it? It only complains if it can be shortened and this can't as the return type is `void`.